docs/color_theming/README.md

```diff
@@ -0,0 +1,58 @@
+# overriding widget colors
```

Comment:
This is A LOT. Having all of these changes will significantly bloat the dracula.sh file.

Reply:
since these options are somewhat scattered in the dracula.sh file, which has been that way for quite some time, i thought it would be sensible to not change any of that, but write easier to read documentation about it. considering that its been a somewhat undocumented feature for a long time, i think that while there probably are quite a few people using these options (including myself), it should've been easier to find them.

Reply:
I agree.